database/PGindex.js

```diff
@@ -10,24 +11,52 @@ const pool = new Pool({
 
 //Postres query SELECT * from ActorInfo INNER JOIN MovieInfo ON movieinfo.ACTORID=actorInfo.ID WHERE movieinfo.movieid=movieid;
 
-const getActorById = (id, callback) => {
-  pool.query(`SELECT * from ActorInfo INNER JOIN MovieInfo ON movieinfo.ACTORID=actorInfo.ID WHERE movieinfo.movieid=${id}`, (err, results) => {
-    if (err) {
-      callback(err);
-    } else {
-      callback(null, results);
-    }
-  });
+const getActorById = async (id) => {
+  const getQuery = {
+    name: 'read-ActorInfo-for-Movie',
+    text: 'SELECT * from ActorInfo INNER JOIN MovieInfo ON movieinfo.ACTORID=actorInfo.ID WHERE movieinfo.movieid=$1',
+    values: [id],
+  }
```

Comment:
Awesome job refactoring this to async/await style and figuring out why the query was taking so long!